### PR TITLE
14- design contact us page

### LINKED ISSUE #14: Level 2: Contact Us Page
## This Issue is Complete When: 

- [ ]  Each contact div is a reloadable component which takes a unique: icon, header, subheader, button

- [ ] button is a reloadable component stored in the "common" folder within the component folder

- [ ] On click, there is an event listener that currently logs "button clicked" on console

## Figma Mockup: 

https://www.figma.com/file/SYCaUO0bX527hsjRt1ZioC/EDD-Help-Page?node-id=24%3A407

## Screenshots: 

Screen Shot 2020-09-27 at 12 14 24 PM

### PR DESCRIPTION
**Pull request by utk0730**
Fixes #14 

- [x] Up to date with 'dev' branch
- [x] Branch name follows issueNumber-issueName syntax
- [x] All PR Status checks are successful
- [ ] Peer reviewed and approved

Question? See the [Contributions guidelines](https://github.com/hannahlivnat/Ask-EDD-Project/blob/main/contributing.md).

### CODE REVIEW

**Review by hannahlivnat — CHANGES_REQUESTED** [review]
Great work on this page, it looks really nice! Can you link the help-center route to the "Contact" link in the nav bar?

**Comment by utk0730** [comment]
> Great work on this page, it looks really nice! Can you link the help-center route to the "Contact" link in the nav bar?

**Comment by utk0730** [comment]
@hannahlivnat This is contact us page right ?
Then shouldn't it be linked from contact link in Navbar ?
Further I realise I have to rename some files , and new page to contact us rather than help center 

Give me a clarity on these

**Comment by hannahlivnat** [comment]
> @hannahlivnat This is contact us page right ?
> Then shouldn't it be linked from contact link in Navbar ?
> Further I realise I have to rename some files , and new page to contact us rather than help center
> 
> Give me a clarity on these

Yes, you can rename help-center route to contact route and it should be linked to "Contact" on the navbar

**Comment by utk0730** [comment]
@hannahlivnat changes done. Please review

File: constants/nav_links.js [hunk 1]
```diff
@@ -31,11 +31,11 @@ export const navLinks = [
   {
     text: "Help Center",
     path: HELP_CENTER,
-    enabled: true,
+    enabled: false,
```
Comment (hannahlivnat) [inline constants/nav_links.js:34]: Help Center link is no longer leading to main page, please fix.

**Review by hannahlivnat — CHANGES_REQUESTED** [review]
contact page link working, but main help center link has been disabled

File: constants/nav_links.js (hunk 1 above)
Comment (utk0730) [reply constants/nav_links.js]: ok,I was confused over which pages should be visible now.
fixed - help center page route.
So we have two route enabled pages - contact page and helpcenter page

Comment (utk0730) [reply constants/nav_links.js]: @hannahlivnat review please

File: components/contact_page.js [hunk 2]
```diff
@@ -1,5 +1,37 @@
-export default function ContactPage() {
-  return <div className="contact-page-wrapper">
-    contact page here
-  </div>
+import style from "../styles/contactusstyle.module.scss";
```
[the remark this answers is not in the document]
Comment (hannahlivnat) [reply components/contact_page.js:1]: @utk0730 added additional review comment here

**Review by hannahlivnat — CHANGES_REQUESTED** [review]

File: components/contact_page.js [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+import { HelpBox } from "../components/common/helpBox";
```
Comment (hannahlivnat) [inline components/contact_page.js:2]: @utk0730 Wanted to double check that you received my comments on contact_page naming a few days ago?

**Review by hannahlivnat — CHANGES_REQUESTED** [review]

File: components/contact_page.js (hunk 3 above)
Comment (utk0730) [reply components/contact_page.js:2]: Sorry, I havn't been keeping well for past 3-4 days.
About changes, You want me to change components name, and also class names right ?
@hannahlivnat

Comment (hannahlivnat) [reply components/contact_page.js:2]: @utk0730 No worries! Yes, everything having to do with the Contact Page (components/classes) should have the contact-page name and everything having to do with the main help center landing page should have the help-center name.

**Review by hannahlivnat — APPROVED** [review]
Looks great, merging in!

**Pull request merged**